tethys/DataReader/GISReader.py

```diff
@@ -44,8 +43,13 @@ def getGISData(settings):
       
     ''' using finer resolution gridded population map (from HYDE or GPW) according to year availability '''
     GISData['pop']          = GetPopData(settings) # dictionary
-     
-    ''' using finer resolution gridded Irrigation map (from HYDE or GMIA) according to year availability '''
+    
+    if settings.UseDemeter:
+        ''' using finer resolution gridded Irrigation map (from demeter by individual crops) according to year availability '''

```

Comment:
Need to change triple quoted strings to comments using the pound sign when not in the docstring of the function.  This change should be made throughout.